docs/src/userguide.md

```diff
@@ -352,12 +352,23 @@ The ordering is consistent with the interpretation that there are infinitessimal
 elements in the algebra; for details see M. Berz, "Automatic Differentiation as
 Nonarchimedean Analysis", Computer Arithmetic and Enclosure Methods, (1992), Elsevier,
 439-450.
+Essentially, the lexicographic order works as follows: smaller order monomials
+are *larger* than higher order monomials; when the order is the same, *larger* monomials
+appear before in the hash-tables; the function [`show_monomials`](@ref).
+```@repl userguide
+x, y = set_variables("x y", order=10);
+
+show_monomials(2)
+```
+Then, the following clearly holds:
```

Comment:
Maybe the "clearly" is pushing it a bit, but this is quite helpful.

Reply:
Done!